tests/test_examples.py

```diff
@@ -19,7 +20,13 @@ def setup():
 
 
 def test_mnist():
-    mnist_test()
+    filename = 'mnist.pkl'
```

Comment:
Mostly a note to myself: A lot of tests are creating files now. In some of the tests I wrote I tested whether these files existed (to make sure our tests don't overwrite people's files unknowingly) and cleaned up the files at the end. Since we're doing this all over the place now I'll try to write a decorator that makes this easier to do.